person.rb

```diff
@@ -1,8 +1,46 @@
-class Person
+# Nameable class
```

Comment:
- [ ] [OPTIONAL] Kindly create separate files for the `Nameable`, `NameableDecorator`, `CapitalizeDecorator`, and `TrimmerDecorator` classes instead of plotting the `person.rb` file with all classes.